### DISCUSSION TITLE
Continously updating

### DISCUSSION THREAD

**Discussion by g3rmanaviator** [qdm12/ddns-updater#338] [Q&A] [answered]
My dynamic IP has not changed in a while, yet the updater keeps writing the same IP address to the updates.json file continously.
The Web UI is telling me my address has "changed" several hundred times, but this appears to only be the same IP address counted as a change.

Any thoughts?

**Comment by g3rmanaviator** [comment, marked as answer]
Looks like the problem is with split DNS. On my internal network the hosts resolved to their RFC1918 address.
When the updater runs it says "Oh, this host resolves to 10.x.x.x, but the IP is a.b.c.d (as per external DNS query), therefore go ahead and update".
I removed the internal split DNS for one of the hosts and this resolved the problem.

One option might be to force DNS lookups from the resolver using a defined set of external DNS servers instead of the built-in OS lookup.

**Reply by g3rmanaviator** [reply]
Problem solved.
Pass the "--dns 8.8.8.8" parameter to the container which forces resolution via external DNS.

**Comment by Matis-11** [comment]
How do I do this with a docker container on a synology, where I do not start it with a command line to add --dns ?

**Reply by qdm12** [reply]
There is a variable I think RESOLVER_ADDRESS (check the readme) which you can use instead.

**Comment by Matis-11** [comment]
Thanks, this was what I needed!
